PhiferTristan/kneel-server
# Sizes View - support of the GET method at route of /sizes.

Created files repository.py, nss_handler.py, and a views folder with sizes_view.py. Added code to json-server.py and added init file to views folder to import SizesView.

## Steps to Test

1. Pull the 'tp-sizes-view-get' branch and switch to it.
2. Seed kneeldiamonds.sqlite3 with the SQL from kneel.sql
3. Verify you have the tables with correct data.
4. Connect to localhost
5. Open Postman, use GET method, and use localhost path with /sizes at end for all sizes, or /sizes/? for a single size and compare with .sqlite3 file to verify correct data is retrieved.
